.github/pull_request_template.md

```diff
@@ -6,7 +6,8 @@
 
 **PR Checklist:**
 
-- [ ] Pre-commit hooks and tests pass (run `scripts/test`)
+- [ ] Pre-commit hooks pass (run `pre-commit run --all-files`)
+- [ ] Tests pass (run `pytest tests --block-network --record-mode=none`)
```

Comment:
Can we make these default so someone can just run `pytest`?

Reply:
yeah good idea